virttest/qemu_vm.py

```diff
@@ -3710,6 +3712,11 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
 
             logging.info("Migrating to %s", uri)
             self.monitor.migrate(uri)
+
+            if mig_inner_cmd:
+                for func in mig_inner_cmd:
+                    func()
+
```

Comment:
Thanks for the comments; I'll rename it to mig_inner_funcs - unless you have a better name in mind?